floss/language/rust/coverage.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+
+import sys
+import logging
+import pathlib
+import argparse
+from typing import List, Tuple, Iterable, Optional
+
+import pefile
+
+from floss.strings import extract_ascii_unicode_strings
+from floss.language.utils import get_extract_stats
+from floss.language.rust.extract import extract_rust_strings
+
+logger = logging.getLogger(__name__)
+
+MIN_STR_LEN = 4
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Get Rust strings")
+    parser.add_argument("path", help="file or path to analyze")
+    parser.add_argument(
+        "-n",
+        "--minimum-length",
+        dest="min_length",
+        type=int,
+        default=MIN_STR_LEN,
+        help="minimum string length",
+    )
+    logging_group = parser.add_argument_group("logging arguments")
+    logging_group.add_argument("-d", "--debug", action="store_true", help="enable debugging output on STDERR")
+    logging_group.add_argument(
+        "-q",
+        "--quiet",
+        action="store_true",
+        help="disable all status output except fatal errors",
+    )
+    args = parser.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+        logging.getLogger().setLevel(logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
+        logging.getLogger().setLevel(logging.INFO)
+
+    try:
+        pe = pefile.PE(args.path)
+    except pefile.PEFormatError as err:
+        logger.debug(f"NOT a valid PE file: {err}")
+        return 1
+
+    path = pathlib.Path(args.path)
+
+    # see only .rdata section
+    buf = path.read_bytes()
+    pe = pefile.PE(data=buf, fast_load=True)
+
+    static_strings = list(extract_ascii_unicode_strings(buf, args.min_length))
+
+    rust_strings = extract_rust_strings(path, args.min_length)
+
+    get_extract_stats(pe, static_strings, rust_strings, args.min_length, 0)
```

Comment:
does the 0 argument make sense here?

Reply:
this is still an open question, we should add least add a comment with an explanation here

Reply:
@Arker123 ^^